notebooks/basics/multiple-systems.ipynb

```diff
@@ -804,59 +842,64 @@
    "id": "4f9e91e9",
    "metadata": {},
    "source": [
-    "#### Independent operations\n",
+    "#### Independent operations and the tensor product\n",
+    "\n",
+    "In the previous sections, we observed that a tensor product state $\\vert \\phi\\rangle \\otimes \\vert \\psi \\rangle$ on a joint system $(\\mathsf{X}, \\mathsf{Y})$ represents independence between the systems, roughly meaning that each system can be described in its own terms. We want to capture a similar notion of independence for operations $M$ and $N$ acting on systems $\\mathsf{X}$ and $\\mathsf{Y}$, respectively. More specifically, we'd like to capture the notion of \"$M$ acts on $\\mathsf{X}$ and $N$ acts on $\\mathsf{Y}$\" in terms of a single operation on the joint system $(\\mathsf{X}, \\mathsf{Y})$. It turns out that this notion can be expressed formally by the tensor product $M \\otimes N$ of the two operations. \n",
+    "\n",
+    "Suppose $(\\mathsf{X}, \\mathsf{Y})$ is in a tensor product state $\\vert \\phi\\rangle \\vert\\psi\\rangle$. If $M$ acts on $\\mathsf{X}$ and $N$ acts on $\\mathsf{Y}$, we should then expect the joint state to change as $M \\vert \\phi \\rangle N\\vert\\psi\\rangle$. Therefore, we define the tensor product operation $M \\otimes N$ to act on tensor product states in precisely this way.\n",
+    "\n",
+    "$$\n",
+    "  (M \\otimes N)\n",
+    "  \\vert \\phi \\rangle \\otimes \\vert \\psi \\rangle \n",
```

Comment:
$(M \otimes N) \vert \phi \otimes \psi\rangle$

(Although there is no other reasonable way to interpret the expression, there's some ambiguity about the order or precedence for tensor products versus matrix products.)